jerry-core/api/jerry-debugger-transport.c

```diff
@@ -161,6 +161,7 @@ jerry_debugger_transport_close (void)
   while (current_p != NULL);
 
   JERRY_CONTEXT (debugger_flags) = JERRY_DEBUGGER_VM_IGNORE;
+  JERRY_CONTEXT (debugger_flags) &= (uint32_t) ~JERRY_DEBUGGER_CONNECTED;
```

Comment:
There is a set (assign) above. Since the flag is not set, there is no point removing it.

Reply:
Yes, right, fixed it. Thank you